nixos/modules/services/misc/paperless.nix

```
@@ -213,9 +213,45 @@ in
       defaultText = literalExpression "pkgs.paperless-ngx";
       description = lib.mdDoc "The Paperless package to use.";
     };
+
+    exporter = {
+
+      enable = mkEnableOption (lib.mdDoc "Enable the paperless document exporter.");
```

```suggestion
      enable = mkEnableOption "automatic paperless document exporter";
```

Perhaps we might want to have a custom description anyways to convey the fact that an export is just an export and not a backup.